core/arch/arm/kernel/boot.c

```diff
@@ -1241,6 +1282,14 @@ static void init_secondary_helper(unsigned long nsec_entry)
 	init_vfp_sec();
 	init_vfp_nsec();
 
+#ifdef CFG_CORE_HALT_CORES_ON_PANIC
+	if (interrupt_add_handler_with_chip(itr_chip,
```

Comment:
I think you can use `interrupt_get_main_chip()` as first argument in the 2 function calls.
I think its would be better than a local varaible under an `#ifdef` directive.